external/source/exploits/CVE-2014-3153/Makefile

```diff
@@ -0,0 +1,17 @@
+
+all: install
+
+build:
+	ndk-build NDK_PROJECT_PATH=. APP_BUILD_SCRIPT=./Android.mk
```

Comment:
As of Dec 2014, GCC 4.8 became the default toolchain for NDK, and it no longer supports the '-mno-thumb' flag.
This should either say explicitly NDK_TOOLCHAIN_VERSION=4.6 or just remove '-mno-thumb' from Android.mk. I tested it with 4.8 and it worked fine.